src/md/community.md

```diff
@@ -4,9 +4,9 @@
 
 The following mailing lists are available for Apex.
 
-- [users@apex.apache.org](http://mail-archives.apache.org/mod_mbox/apex-users/) is a discussion forum for people who want to build applications and/or solutions using Apex. [subscribe](mailto:users-subscribe@apex.apache.org?subject=send%20this%20email%20to%20subscribe), [unsubscribe](mailto:users-unsubscribe@apex.apache.org?subject=send%20this%20email%20to%20unsubscribe), [ponymail](https://lists.apache.org/list.html?users@apex.apache.org), [nabble](https://s.apache.org/apex-users), [archives](http://mail-archives.apache.org/mod_mbox/apex-users/)
-- [dev@apex.apache.org](http://mail-archives.apache.org/mod_mbox/apex-dev/) is for people who want to contribute code to Apex. Traffic on the list is generally high and includes JIRA notifications and code review comment. [subscribe](mailto:dev-subscribe@apex.apache.org?subject=send%20this%20email%20to%20subscribe), [unsubscribe](mailto:dev-unsubscribe@apex.apache.org?subject=send%20this%20email%20to%20unsubscribe), [ponymail](https://lists.apache.org/list.html?dev@apex.apache.org), [nabble](https://s.apache.org/apex-dev), [archives](http://mail-archives.apache.org/mod_mbox/apex-dev/)
-- [commits@apex.apache.org](http://mail-archives.apache.org/mod_mbox/apex-commits/) is for commit messages and patches to Apex. [subscribe](mailto:commits-subscribe@apex.apache.org?subject=send%20this%20email%20to%20subscribe), [unsubscribe](mailto:commits-unsubscribe@apex.apache.org?subject=send%20this%20email%20to%20unsubscribe), [ponymail](https://lists.apache.org/list.html?commits@apex.apache.org), [nabble](https://s.apache.org/apex-commits), [archives](http://mail-archives.apache.org/mod_mbox/apex-commits/)
+- [users@apex.apache.org](https://lists.apache.org/list.html?users@apex.apache.org) is a discussion forum for people who want to build applications and/or solutions using Apex. [subscribe](mailto:users-subscribe@apex.apache.org?subject=send%20this%20email%20to%20subscribe), [unsubscribe](mailto:users-unsubscribe@apex.apache.org?subject=send%20this%20email%20to%20unsubscribe), [ponymail](https://lists.apache.org/list.html?users@apex.apache.org), [nabble](https://s.apache.org/apex-users), [archives](https://mail-archives.apache.org/mod_mbox/apex-users/)
```

Comment:
Yes, but IMO, it is better to have all 3 explicitly spelled out at the end. A user may be looking specifically for the ponymail link and it is not clear that the default link points to the ponymail unless the link is followed.